matching/match.py

```diff
@@ -168,16 +172,17 @@
                     "passnum", "match_strict", "match_moderate",
                     "match_relaxed", "match_review", "weight"]
     for comp_vars in config["comp_names_by_pass"]:
-        for var in comp_vars:
+        for var in config["comp_names_by_pass"][comp_vars]:
             if var not in output_vars:
                 output_vars.append(var)
 
     # Run match in parallel
     all_cands = []
-    tot_pass_cnt = len(config['blocks_by_pass'])
+    passes =  sorted(config['blocks_by_pass'])
     last_time = time.time()
     i = 0
-    for passnum in range(tot_pass_cnt):
+    for passnum in passes:
```

Comment:
Same (I think I mentioned that in the description). Once I settle all the other changes, I can do that. Do we also want to rename the field in outputs?

Reply:
Sorry for missing your note in the description! Glad we are on the same page. I think renaming the output field is good. We can deal with any downstream changes as we run into them. 

I think it would be good to do this in a separate PR for easier review!